runtime/onnion_runtime/dropout.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+
+class Dropout:
+    def __init__(self, opset_version, **kwargs):
+        self.version = opset_version
+        self.seed = kwargs.get("seed")
+        self.consumed_inputs = kwargs.get("consumed_inputs")
+        self.is_test = kwargs.get("is_test", 0)
+        self.ratio = kwargs.get("ratio", 0.5)
+
+    def run(self, data, ratio=None, training_mode=None):
+        if self.version >= 12:
+            if ratio is not None:
+                self.ratio = ratio
+            if training_mode is not None:
+                return dropout(data, self.ratio, self.seed, training_mode, return_mask=True)
```

Comment:
Does onnion support training?
If not, you can assert training_mode=None.

If onnion supports trainning, this code may compare Tensor with boolean in L29.
I think some conversion like https://github.com/onnx/onnx/blob/91bf3f52c96919af8de4d90c4e70316a47112178/onnx/backend/test/case/node/dropout.py#L108-L109 is necessary.

Reply:
onnion supports training and fix it.